danuluma/dannstore
# #161214014 add endpoint to get all sales records

#### What does this PR do?
Add an endpoint to get all sales records

#### Description of Task to be completed?
Add GET /sales endpoint
Write a test for it

#### How should this be manually tested?
* Set up the app locally as described in the installation documentation
* Use postman to send a GET request to http://localhost:5000/dann/api/v1/sales 
* You should get a response containing the following ```{"Error": "There are no sale records"}```. This is because there are currently no sales records


#### What are the relevant pivotal tracker stories?
[#161214014](https://www.pivotaltracker.com/story/show/161214014)
